main/manager.cc

```diff
@@ -2524,9 +2497,11 @@ void UpdateSystemCB(XtPointer client_data, XtIntervalId *time_id)
     {
         if (LastDay != -1)
```

Comment:
is this code still needed? I don't quite know if it is only for LicenseCheck or other parts of the program